wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,212 @@
-| Select (`1`) | **Left-click/drag:** Select/Adjust the notes/sliderpoints location. **Right-click:** Remove notes/sliderpoints. **`Ctrl` + Click:** Multiple selection. | Select and modify existing notes. |
-| Circle (`2`) | **Left/Right-click:** Add/Remove hit circle. | Hit Circle. |
-| Slider (`3`) | **Left/Right-click:** Start/End Slider. Add/Remove sliderpoints (white square). **Left-Click x2:** New curve section (red square). | Slider. |
-| Spinner (`4`) | **Left/Right-click:** Start/End Spinner at current timestamp. | Spinner. Make sure Auto can at least score 2,000 Bonus per Spinner (to prevent the infamous spinner spam/ninja spinner). |
+| `Sampleset` | Auto, Normal, Soft, Drum | This ignores the timing section's sampleset setting, and adds the selected one. |
+| `Additions` | Auto, Normal, Soft, Drum | Additions hitsound means add Clap, Finish, and Whistle with the selected set. |
+| `Select` (`1`) | `Left click`**/drag**: Select/Adjust the objects/sliderpoints location. `Right click`: Remove objects/sliderpoints. `Ctrl` + `Click`: Multiple selection. | Select and modify existing hit objects. |
+| `Circle` (`2`) | `Left click`: Add object. | |
+| `Slider` (`3`) | `Left/Right click`: Start/End slider. Add/Remove sliderpoints. **Double** `Left click`: New curve section. | |
+| `Spinner` (`4`) | `Left/Right click`: Start/End spinner at current timestamp. | |
 
-### Centre (Playfield)
+### Playfield
 
-**Visual representation of mapping based on current timestamp.** Map your design here and your design will appear exactly like you mapped during play. This is WYSIWYG (what you see is what you get) mapping. For text-based (usually for fine-tuning) mapping, you can open the `.osu` file directly using Notepad.
+![](img/playfield.jpg "Visual representation of the current timestamp")
 
-### Centre-Right (Hitsounds & Assist Tools)
+Difficulties are mapped here and during gameplay, they will appear exactly like displayed in the editor. For fine-grained mapping, the `.osu` file can be directly edited using a text editor like Notepad.
 
-**List of available subtools complement to the original DIY toolkit.**
+### Subtools
 
-![Hitsounds & Assist Tools](img/BE_HSAT.jpg "Hitsounds & Assist Tools")
+![](img/subtools.jpg "List of available subtools")
 
-| Button (Keyboard Shortcut) | Usage | Description |
+| Button (Keyboard shortcut) | Usage | Description |
 | :-- | :-- | :-- |
-| New Combo (`Q`) | **Right-click:** Change current note to next combo colour. | Note combo. This is equivalent to a verse / measure in song / instrument script. An acceptable range is about 5-20. |
+| `New Combo` (`Q`) | `Right click`: Change current object to next combo colour. | Hit object combo. This is equivalent to a verse in song script. An acceptable range is about 5–20. |
```

Comment:
the right click action only works when you are in object placement mode; if a regular object is right-clicked, it's removed

Reply:
(also, this merits a link to `/wiki/Beatmapping/Combo`)

Reply:
Changed to this:

![image](https://github.com/ppy/osu-wiki/assets/58957468/ce8eb052-4c7d-4eb0-a63f-59044fa7bfbb)